File: python/PROBLEMS/1037.py
e = float(input())
if e >= 0:
    if e <= 25:
        print("Intervalo [0,25]")
    elif e <= 50:
        print("Intervalo (25,50]")
    elif e <= 100:
        print("Intervalo (75,100]")
    else:
        print("Fora de intervalo")
else:
    print("Fora de intervalo")

File: c/PROBLEMS/1037.c
#include <stdio.h>

int main(){
    double e;
    scanf("%lf", &e);
    if (e >= 0){
        if (e <= 25){
            printf("Intervalo [0,25]\n");
        }
        else if (e <= 50){
            printf("Intervalo (25,50]\n");
        }
        else if (e <= 100){
            printf("Intervalo (75,100]\n");
        }
        else{
            printf("Fora de intervalo\n");
        }
    } else{
        printf("Fora de intervalo\n");
    }
}